Bottom: d4bd1b1090e317531883ef00eb0b9f194deb0d1e
Top:    22f79f23bb812fa1bfa6ab6c0c32852bcd0d920e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 13:39:06 +0100

Fix last commit to work with new crosshair notification API


---

diff --git a/src/action.c b/src/action.c
index 7adb9de..4181f2e 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4847,7 +4847,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
   char *function = ARG (0);
   if (function && PCB->ElementOn)
     {
-      HideCrosshair (true);
       switch (GetFunctionID (function))
 	{
 	case F_Object:
@@ -4897,7 +4896,6 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	      }
 	  }
 	}
-      RestoreCrosshair (true);
     }
   return 0;
 }
